controllers/users.js

```diff
@@ -5,3 +5,32 @@ const jwt = require("jsonwebtoken")
 
 
 db.on('error', console.error.bind(console, 'MongoDB connection error'))
+const SALT_ROUNDS = 11;
+const TOKEN_KEY = "thebestgroupever";
+const signUp = async (req, res) => {
```

Comment:
isn't the token key "securetoken" ?

Reply:
you guys can decide to change it

Reply:
It has to be the same as the other token that is already established.